### PR TITLE
Added properties of bodies and fixed an error in "area_of_circle"

### LINKED ISSUE #14: Adding tools for calculating properties of sections
Would it be appropriate to add tools for calculating properties of sections? I already have written for my personal use calculators for calculating section modulus, centroid, and second moment of inertia for certain shapes. If you feel these don't belong in MathTools I won't spend the time to clean them up for use.

### PR DESCRIPTION
**Pull request by MrRobotMan**
I added a file to the geometry tools to calculate section modulus (and other properties) of bodies. These include rectangular bars, round bars, T sections, and I sections. 

Lastly I noticed that the area_of_circle function allowed for complex numbers but was not correctly calculating the area of a circle in the complex plane. The correct radius for a complex radius is the complex modulus of the radius (i.e. radius = a + bi = [a^2 + b^2]^(1/2)).

### CODE REVIEW

File: mttools/geometry_tools/__init__.py [hunk 1]
```diff
@@ -5,7 +5,15 @@
 from math import sqrt, pi
 from typing import List
 
-from mttools.utils.types import Number
+from mttools.utils.types import Number, RealNumber
+from mttools.utils.exceptions import DimensionError
+from .PropertiesOfBodies import (
```
Comment (KGB33) [inline mttools/geometry_tools/__init__.py]: I would like users to import the functions from the `PropertiesOfBodies` file. This adds more information when importing and follows the same practice and the rest of the `foo_tools` sections.

**Review by KGB33 — CHANGES_REQUESTED** [review]
Some of your doctests are failing.  You can use 
```
python -m doctest mttools/geometry_tools/PropertiesOfBodies.py -o NORMALIZE_WHITESPACE
```

To run the doctests for your file. 

Also, are there any properties you could test for?

**Comment by KGB33** [comment]
Just a heads up there's a problem with `black`'s stable branch that the new CodeQuality action I added. If it's not fixed by the time you resubmit that'll be fine I'll merge it anyways.

File: mttools/geometry_tools/PropertiesOfBodies.py [hunk 2]
```diff
@@ -0,0 +1,240 @@
+from math import pi
+from typing import Tuple
+
+from mttools.utils.types import RealNumber
+
+
+def rect_bar(depth: RealNumber, thickness: RealNumber) -> dict:
```
[the remark this answers is not in the document]
Comment (MrRobotMan) [reply mttools/geometry_tools/PropertiesOfBodies.py]: Done. The centroid is now 2 keys, Centroid X and Centroid Y

**Comment by MrRobotMan** [comment]
> 
> 
> Some of your doctests are failing. You can use
> 
> ```
> python -m doctest mttools/geometry_tools/PropertiesOfBodies.py -o NORMALIZE_WHITESPACE
> ```
> 
> To run the doctests for your file.
> 
> Also, are there any properties you could test for?

What do you mean about properties I can test for? Do you mean other properties of the section itself?

**Comment by KGB33** [comment]
> What do you mean about properties I can test for? Do you mean other properties of the section itself?

Is there anything that's true for every possible body? 

For example, for every triangle, the angles always add up to 180degrees.

**Comment by MrRobotMan** [comment]
I can't think of anything that could be checked this way. All of these are just cross-sections of solid bodies.

**Comment by KGB33** [comment]
Awesome! Thanks for the PR

**Pull request merged**